packages/fxa-content-server/app/scripts/templates/pair/unsupported.mustache

```diff
@@ -1,14 +1,33 @@
-<div class="card">
+<div class="card-base">
   <header>
-    <h1 id="pair-unsupported-header" class="card-header">{{#t}}Pair using an app{{/t}}</h1>
+    {{#isDesktopNonFirefox}}
+      <h1 class="mb-5 text-grey-400 text-base">{{#t}}Connect another device{{/t}}</h1>
+    {{/isDesktopNonFirefox}}
+    {{^isDesktopNonFirefox}}
+      <h1 id="pair-unsupported-header" class="card-header">{{#t}}Pair using an app{{/t}}</h1>
+    {{/isDesktopNonFirefox}}
   </header>
 
   <section>
     <div class="error"></div>
 
-    <div class="bg-image-pair-fail"></div>
+    {{#isDesktopNonFirefox}}
+      <div class="bg-no-ff-desktop bg-contain bg-center bg-no-repeat h-44" aria-label="{{#t}}Pink exclamation mark denoting Firefox is missing from a desktop device{{/t}}" role="img"></div>
+      <h2 id="pair-unsupported-header" class="card-header focus:outline-none mt-5" tabindex="-1">{{#t}}Oops! It looks like you’re not using Firefox.{{/t}}</h2>
+      <div class="flex bg-orange-50 px-4 py-3 mt-6 rounded-lg mb-4">
+        <div class="bg-icon-warning bg-contain bg-center bg-no-repeat w-6"></div>
+        <p class="ps-3">{{#t}}Switch to Firefox and open this page to connect another device.{{/t}}</p>
+      </div>
+      <div class="flex">
+        <a id="download-firefox" href="https://www.mozilla.org/firefox/new/" target="_blank" class="cta-primary cta-xl">{{#t}}Download Firefox{{/t}}<span class="sr-only">{{#t}}Opens in new window{{/t}}</span></a>
```

Comment:
What is `sr-only` stand for? Is that an accessibility note to explain that the link opens in a new window?

Reply:
Correct, `sr-only` is a class we have [in Tailwind](https://tailwindcss.com/docs/screen-readers) that's visually hidden from users but still read to screenreaders. We mostly use it in our [React LinkExternal component](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/components/LinkExternal/index.tsx#L43), but the page in this PR is still in Backbone and we don't have a component for it there, so I manually added it.